include/mega/command.h

```diff
@@ -86,7 +91,9 @@ class MEGA_API Command
     const char* getstring() const;
 
     Command();
-    virtual ~Command() { }
+    virtual ~Command() = default;
+
+    MEGA_DISABLE_COPY_MOVE(Command)
```

Comment:
Having looked at this a bit closer, there are no owning pointers etc in this class that would cause problems should one of these be copied.   Should we really disallow something that's not problematic?  There could be legitimate uses for it, and in fact there could be third party clients relying on that somehow.

Reply:
I think we should disallow because copying/moving `levels` doesn't work by default. So either we implement explicit semantics or disallow. What do you think?

Reply:
After IRL realized that `levels` is fine to copy/move. Agreed IRL to default all special member functions.